contracts/SB_scripts/mainnet/Main0026.sol

```diff
@@ -0,0 +1,22 @@
+/* Recreate loan products on mainnet (fix for discountRate rounding error) */
```

Comment:
"Issue legacy tokens in order to repay two loans (id 23 & 24) in legacy loanManager contract at `0xCBeFaF199b800DEeB9EAd61f358EE46E06c54070`. 
The ETH value for the issued A-EUR is going to be sent into Reserve contract at 0x65F30f8DD20c707C1938CcAd7416c7381E6eB9C8 "